nemo/collections/tts/models/hifigan.py

```diff
@@ -175,10 +183,11 @@ def training_step(self, batch, batch_idx, optimizer_idx):
             "global_step": self.global_step,
             "lr": self.optim_g.param_groups[0]['lr'],
         }
-        self.log_dict(metrics, on_step=False, on_epoch=True, sync_dist=True)
+        self.log_dict(metrics, on_step=True, sync_dist=True)
```

Comment:
on_step seems to be too frequent. Or will lightning automatically log at the log_every_n_steps interval?

Reply:
automatically